nailgun/entities.py

```diff
@@ -1096,6 +1135,23 @@ def __init__(self, server_config=None, **kwargs):
         }
         super(ContentView, self).__init__(server_config, **kwargs)
 
+    def read(self, entity=None, attrs=None, ignore=()):
+        """Fetch an attribute missing from the server's response.
+
+        For more information, see `Bugzilla #1237257
+        <https://bugzilla.redhat.com/show_bug.cgi?id=1237257>`_.
+
+        """
+        if (getattr(self._server_config, 'version', parse_version('6.1')) <
```

Comment:
It would be better if this sort of comparison could be made:

```python
from packaing.version import parse
...
if self._server_config.version < parse('6.1'):
    pass  # some workaround
```

However, this will only work if `version` is guaranteed to exist on `self._server_config`. Making that guarantee is tricky to get right. One option is to simply require that the user provide a version any time they make a `ServerConfig` object:

```python
cfg = ServerConfig('example.com', version='6.1')
```

This is straightforward, but does break backward compatibility and will render many saved server configs unusable. Another option is to provide a default value.

```python
>>> from packaging.version import Version
>>> cfg = ServerConfig('example.com')
>>> cfg.version == Version('1!0')  # epoch 1, release 0.0.0
True
```

The default version would be set to "epoch 1, release 0.0.0", which would *always* be greater than any version in epoch 0:

```python
>>> from packaging.version import Version
>>> Version('1!0') > Version('0.0.0')
True
>>> Version('1!0') > Version('0.0.1')
True
>>> Version('1!0') > Version('0.1.0')
True
>>> Version('1!0') > Version('1.0.0')
True
>>> Version('1!0') > Version('6.0.8')
True
```

...well, so long as Satellite never changes its versioning scheme:

```python
>>> from packaging.version import Version
>>> Version('1!0') > Version('1!0.0.1')
False
```